corehq/apps/app_manager/suite_xml/xml_models.py

```diff
@@ -540,12 +545,12 @@ class Entry(OrderedXmlObject, XmlObject):
     datums = NodeListField('session/datum', SessionDatum)
     queries = NodeListField('session/query', RemoteRequestQuery)
     session_children = NodeListField('session/*', _wrap_session_datums)
+    local_datums = NodeListField('session/*[self::datum or self::instance-datum]', _wrap_session_datums)
```

Comment:
I don't love adding this - it seems excessive to have the three individual attributes, then `session_children` to combine all three, then `local_datums` to combine just  datum and instance-datum - but I didn't find a better way to do it.

Reply:
I can't think of a better way either, maybe @snopoke?

Reply:
I guess there are 2 alternatives:

1. add an `instance_datums` field:   `for datum in entry.datums + entry.instance_datums:`
2. use `session_children` and do a type check: `if isinstance(child, (SessionDatum, InstanceDatum)`

I'm not sure that either is better than the current solution. (1) has the advantage of providing more granular access and being more similar to `RemoteRequestSession`

I'm also not sure about the name `local_datums`. I'm assuming that's in contrast to `remote_datums`? I might prefer `all_datums`.

Reply:
Updated the naming in d2178cb638f - it was to contrast with `remote_datums`, yes, but yeah, "local" doesn't really make sense.

> for datum in entry.datums + entry.instance_datums:

I originally did this and found it surprisingly challenging to join the two NodeListFields.